services/ui-src/.env

```diff
@@ -1,8 +1,11 @@
+LOCAL_LOGIN=true
```

Comment:
My mistake was in adding these local config values to ui-src/.env. If you take them out you should be able to avoid having to set them explicitly to false and "" when running in CI.